packages/webapp/src/views/credits-purchase/credits-terms-and-conditions.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'form-layout.html' %}
+
+{% set serviceName = "Purchase statutory biodiversity credits" %}
+{% set pageHeading = "Terms and Conditions for the purchase of Statutory Biodiversity Credits" %}
+
+{% block formContent %}
+    <div class="govuk-body">
+        {% if err %}
+            {% include "errorbanner.html" %}
+        {% endif %}
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+              <h1 class="govuk-heading-xl">{{ pageHeading }}</h1>
+                <p>The statutory biodiversity credit scheme is run by Natural England on behalf of the Department for Environment, Food and Rural Affairs (Defra) as part of the Environment Act 2021.</p>
+                
+                <h2 class="govuk-heading-m">Privacy notice</h2>
+                <p class="govuk-body">Our <a href="#">privacy notice </a>sets out how we use the personal data.</p>  
+                
+                <h2 class="govuk-heading-m">Your responsibilities</h2>
+                <p class="govuk-body">It is your responsibility to ensure:</p>
+                <ul class="govuk-list govuk-list--bullet">
+                  <li>you have purchased the correct number, and tier, of statutory biodiversity credits to meet your BNG obligation for development under the Planning Permission referenced in your application</li>
+                </ul>
+                
+                <h2 class="govuk-heading-m">Anti-money laundering</h2>
+                <p class="govuk-body">All applications to purchase statutory biodiversity credits are subject to anti-money laundering checks. They can only be purchased by applicants who pass those checks.</p>
+                
+                <h2 class="govuk-heading-m">Purchase terms</h2>
+                <p class="govuk-body">You have no obligation to Natural England to buy the statutory biodiversity credits in this service. The terms and conditions for purchase of statutory biodiversity credits will apply if you proceed.</p>
+                
+                <h2 class="govuk-heading-m">Introduction</h2>
```

Comment:
This heading should be at the top, and the paragraph below is a repeat of what is at the top